### ISSUE TITLE
Radar no longer works

### ISSUE THREAD

**Issue by dsdude123** [dsdude123/StarBot#41]
Looks like the old method of getting radar doesn't work anymore.

```
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /WxImages/RadarLoop/tiw_None_anim.gif was not found on this server.</p>
</body></html>
```

**Comment by dsdude123** [comment]
https://s.w-x.co/staticmaps/wu/wxtype/county_loc/tiw/animate.png

**Comment by dsdude123** [comment]
Resolved by 0258b7b3969b7ff69100fb6ef37dd417c9e464d8
